src/features/Rule/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { Dialog, Transition } from "@headlessui/react";
+import React, { Fragment, useEffect, useState } from "react";
+import { AiOutlineClose } from "react-icons/ai";
+
+const Rule = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  // OPEN RULE-MODAL: USER'S FIRST VISIT
+  useEffect(() => {
+    const hasPlayed = localStorage.getItem("hasPlayed");
+
+    if (!hasPlayed) {
+      setTimeout(() => {
+        setIsOpen(true);
+        localStorage.setItem("hasPlayed", "true");
```

Comment:
This is redundant.

Reply:
Thanks, I removed it.
https://github.com/mikuweb/Pokemon-App/pull/3/commits/3df8c5334b5ebb4dd80ecc1de26bc271b6ef8efc